src/common/SendChannel.ts

```diff
@@ -0,0 +1,132 @@
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Generates a random or pseudo-random string identifier.
+ *
+ * @returns {string}
+ */
+function uniqueId() {
```

Comment:
why not just `uuidv4`?

Reply:
Alternatively, `import { v4 as uniqueId } from 'uuid';`

Reply:
The message id is really only useful for debugging, so I suspect it was just an easy way to test with a stable id (rather than relying on the internal detail of the uuid library.)
But no tests use it, so might as well retire it!